tests/integration/regions/test_plugin_region_table.py

```diff
@@ -11,13 +12,16 @@ def exec_test_command(args: List[str]):
     )
     return process
 
-
 def test_output():
     process = exec_test_command(BASE_CMD)
     output = process.stdout.decode()
     lines = output.split("\n")
-    lines = lines[3 : len(lines) - 2]
+    lines = lines[3: len(lines) - 2]
     for line in lines:
-        assert "-" in line
-        assert "✔" in line
-        assert "|" in line
+        check_marks = re.findall("✔", line)
+        hyphens = re.findall("-", line)
+        pipes = re.findall(r"\|", line)
+
+        assert all(mark in line for mark in check_marks)
+        assert all(hyphen in line for hyphen in hyphens)
+        assert all(pipe in line for pipe in pipes)
```

Comment:
Nice work! I wonder why we have to loop through `check_marks` if it’s all `✔`?

Reply:
The lines are horizontal, i believe this is checking to see if there are checks, hyphens, and pipes on every line which is probably safe until we hit a datacenter capable of every possible feature.

Reply:
But it looks like we also to assert mark in the line if we found the mark in the line, which is kind of weird..

Reply:
It was taking the column outer line as well for the asserts which was like this - ┡━━━━━━━╇━━━━━━━╇━━━━━╇━━━━━━╇━━━━━╇━━━━╇━━━━━╇━━━━╇━━━━╇━━━━━╇━━━━━━╇━━━━━━━╇━━━━━━╇━━━━━━╇━━━━━━━┩'